v2/snmp/serverhooks.go

```diff
@@ -54,10 +54,10 @@ var DiagnosticServerHooks = &ServerHooks{
 		log.Printf("Error err:%v\n", err)
 	},
 	WriteComplete: func(config *serverConfig, addr net.Addr, output []byte, err error) {
-		log.Printf("WriteComplete target:%s err:%v data:%s\n", addr, err, hex.EncodeToString(output))
+		log.Printf("WriteDone target:%s err:%v data:%s\n", addr, err, hex.EncodeToString(output))
```

Comment:
Name WriteDone/WriteComplete

Reply:
Reverted unintended changes to serverhooks.